tests/pipeline_test.py

```diff
@@ -1,11 +1,6 @@
-import os
-import sys
-
-#################################################################
-# import internal src 
-src_path = os.path.join(os.path.dirname(__file__), '..', 'src')
-sys.path.append(src_path)
```

Comment:
Removing the path append here causes the failed on test. Is there a reason to remove this?